.github/ISSUE_TEMPLATE/NEW_EXPERIMENT.md

```diff
@@ -0,0 +1,85 @@
+---
+name: "\U0001F4CB New experiment proposal"
+about: "Proposal and workflow guideline for adding a new experiment \U0001F52C."
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- ⚠️ If you do not respect this template, your issue will be closed -->
+<!-- ⚠️ Make sure to browse the opened and closed issues -->
+
+# New experiment proposal
+Provide the experiment and implementation details below and ask the qiskit-experiments core team to review 
+your proposal. 
+
+## General details
+
+### Experiment name
+What is the experiment class name? This name will also be used in the API documentation and tutorial.
+
+### Experiment type
+What is the experiment type? Characterization, calibration, verification, validation, or other? 
+
+### Experiment protocol
+Provide a concise description of the experiment. Make sure you cover the following aspects:  
+* What is the main goal of the experiment?
+* Describe the circuits/pulses and their main parameters needed for the experiment
+* What are the main outputs of the experiment?
+
+### Experiment analysis
+Provide a concise description of the experiment's analysis. Make sure you cover the following aspects:
+* What is the main fit model for the experiment? 
+* What are the main fit parameters? Specify parameter defaults and bounds where relevant
+* How would you evaluate whether the fit is good or bad?
+
+### References 
+If applicable, provide a few of the most relevant references here.
+
+## Implementation details
+Provide additional details pertaining to the implementation of the experiment and its analysis. Use the following as 
+guiding points. Provide code snippets and usage examples where appropriate. 
+### Experiment implementation
+* What are the base classes for the experiment and its analysis?
+* List the input parameters, required and optional, and types of each
+* Experiment options and default values (e.g. default transpile and run options)
+* Are there any limitations or special requirements for running the experiment as part of a composite experiment? 
+Provide details.
+
+### Experiment analysis
+* Analysis options and default values
+* How does it generate initial guesses
+* What plots will be generated? 
+
+---
+
+# Workflow
```

Comment:
Basically everything in this section relates to the PR that the author needs to submit implementing the experiment after the proposal is approved. Perhaps this section could be moved to a "New Experiment Pull Request" template instead of this new experiment  issue template. If you follow this suggestion, I would link to the pull request template here at the bottom of the issue template.

Reply:
I think there is an advantage in having everything in one place. One of those being that the authors would see right away all the requirements for adding a new experiment. And they can obviously copy the task list to their PR. I'd say let's keep it this way and we can modify later if it turns out to be a hassle or not being followed in the PRs